src/components/button/button.tmpl.js

```diff
@@ -0,0 +1,15 @@
+import { Component } from "../utils";
+
+const context = {
+  value: '',
+  class: ' '
+};
+
+
+const blockTemplate = function() {
+
+return `<button class="{{class}} button" >{{value}}</button>`
+}
```

Comment:
Тут кажется поехало форматирование:

```suggestion
const blockTemplate = function() {
    return `<button class="{{class}} button" >{{value}}</button>`;
};
```

Reply:
Для таких однострочных выражений, удобно использовать стрелочные функции:

`const blockTemplate  = () => '<button class="{{class}} button" >{{value}}</button>';`